go/engine/kex2_provisionee.go

```diff
@@ -288,6 +290,21 @@ func (e *Kex2Provisionee) handleDidCounterSign(sig []byte, sdhBoxes []keybase1.S
 		return err
 	}
 
+	// remember how to update a user with the new links
+	e.localDelegateFn = func(u *libkb.User) (err error) {
+		// Signing sibkey link
+		u.SigChainBump(decSig.linkID, decSig.sigID)
+		err = u.LocalDelegateKey(e.eddsa, decSig.sigID, e.eddsa.GetKID(), true /*isSibkey*/, false /*isEldest*/)
```

Comment:
bah, I think these are wrong. Passing delegated KID in the place of signing KID.